Python/Product/Miniconda/Miniconda3-x64/Lib/_nsis.py

```diff
@@ -132,7 +187,7 @@ def run_post_install():
 
 
 allusers = (not exists(join(ROOT_PREFIX, '.nonadmin')))
-out('allusers is %s\n' % allusers)
+# out('allusers is %s\n' % allusers)
```

Comment:
i wonder why this was commented out

Reply:
that was probably just logging, not sure why they would take it out though